lib/assets/javascripts/twine.coffee

```diff
@@ -124,6 +121,18 @@
 
     Twine
 
+  # IE and Safari don't support node.children for DocumentFragment and SVGElement nodes.
+  # If the element supports children we continue to traverse the children, otherwise
+  # we stop traversing that subtree.
+  # https://developer.mozilla.org/en-US/docs/Web/API/ParentNode.children
+  # As a result, Twine are unsupported within DocumentFragment and SVGElement nodes.
+  #
+  # We also prevent nodes from being iterated over more than once by locking
```

Comment:
"locking" is an odd term that makes it sound like something special is happening .  I'd expect something like, `A static list of current DOM nodes is iterated over, preventing immediately inserted/ deleted siblings from causing double/ missed binds and unbinds`.

Reply:
"caching a copy" would be clear to me.